include/mbedtls/aes.h

```diff
@@ -130,12 +130,10 @@ int mbedtls_aes_crypt_ecb( mbedtls_aes_context *ctx,
 
 #if defined(MBEDTLS_CIPHER_MODE_CBC)
 /**
- * \brief This function performs the operation defined in the \p mode parameter (encrypt/decrypt), on the AES-CBC input data buffer defined in the \p ctx parameter. 
- * It can be called as many times as needed, until all the input data is processed.\par
- * mbedtls_aes_init(),  mbedtls_aes_setkey_enc() or mbedtls_aes_setkey_dec() must be called before 
+ * \brief This function performs the operation defined in the \p mode parameter (encrypt/decrypt), on the AES-CBC input data buffer defined in the \p input parameter. 
```

Comment:
same comment:
I don't think we should call it `AES-CBC input data buffer`.
It is enough to name it input data buffer.
AES-CBC is the cipher mode, not the buffer type